.github/workflows/publish.yml

```diff
@@ -55,6 +55,33 @@ jobs:
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
           command: pages publish .svelte-kit/cloudflare --project-name hollama
 
+  electron:
+    needs: release
+    if: ${{ needs.release.outputs.new_release_published == 'true' }}
+
+    name: Build & release Electron (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-13, ubuntu-latest, windows-latest] # AFAIK macos-13 is the lastest one on x86
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+
+      - name: Install Electron & SvelteKit's dependencies
+        run: npm ci
+
+      - name: Build & release Electron app
+        uses: coparse-inc/action-electron-builder@v1.0.0
+        with:
+          github_token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          build_script_name: 'electron:build'
+          release: true
+        env:
+          ADAPTER: electron-node
```

Comment:
I updated `build_script_name` so it runs the script you created:
```json
"electron:build": "ADAPTER=electron-node vite build && electron-builder"
```

However, I'm also intentionally adding the `ADAPTER` environment variable in this file because `window-latest` doesn't allow you to add environment variables as part of a command.